test/components/views/messages/DateSeparator-test.tsx

```diff
@@ -64,7 +64,11 @@ describe("DateSeparator", () => {
 
     beforeEach(() => {
         global.Date = MockDate as unknown as DateConstructor;
-        (SettingsStore.getValue as jest.Mock).mockReturnValue(true);
+        (SettingsStore.getValue as jest.Mock) = jest.fn((arg) => {
+            if(arg === UIFeature.TimelineEnableRelativeDates) {
+                return true;
+            }
+        });
```

Comment:
We now use `UIFeature.TimelineEnableRelativeDates` `"feature_jump_to_date"` in `src/components/views/messages/DateSeparator.tsx` so we need to be more specific about which one should be enabled now.